src/config.cpp

```diff
@@ -19,11 +19,17 @@
  *
  */
 #include "internal.h"
+#include "qregularexpression.h"
 
 #include <stdlib.h>
 
 #include <QDebug>
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
 #include <QXmlDefaultHandler>
+#else
+#include <QXmlStreamReader>
```

Comment:
Do we really need different classes used on Qt5 and Qt6? The QXmlDefaultHandler seems still available in core5compat and QXmlStreamReader should be available in Qt5?

Reply:
Don't like and use compat layer.